releasenotes/notes/c-api-circuit-3ecb43d396425bed.yaml

```
@@ -0,0 +1,5 @@
+---
+features_circuits:
+  - |
+    The C API for Qiskit has been extended with support for building and
+    interacting with quantum circuits.
```

Maybe we should mention that the support of this API is limited ?
```suggestion
    The C API for Qiskit has been extended with limited support for building and
    interacting with quantum circuits.
```

---

I'm not actually sure what limited would mean here. The API lets you build and interact with a quantum circuit. It's not the same as the python API but there will be continued incremental improvements to the API over time. We're also 3 months out from the final 2.1.0 release so it's hard to say how limited the API will be when we finish.